inputs/prod_customize.json

```diff
@@ -0,0 +1,7 @@
+{
+    "disable_plugins": [
+    ],
+
+    "enable_plugins": [
+    ]
+}
```

Comment:
It is, but it could easily be shipped in the package and since the lists are empty it would effectively not alter anything by default. But if it is preferred not to ship it, then that's fine too.